packages/twenty-front/src/modules/object-record/utils/computeRecordBoardColumnDefinitionsFromObjectMetadata.ts

```diff
@@ -34,11 +35,25 @@ export const computeRecordBoardColumnDefinitionsFromObjectMetadata = (
     actions: [
       {
         id: 'edit',
-        label: 'Edit from settings',
-        icon: IconPencil,
+        label: 'Edit',
+        icon: IconSettings,
         position: 0,
         callback: navigateToSelectSettings,
       },
+      {
```

Comment:
Are we sure we're not duplicating something here ?

Reply:
No no, I wanted to confirm if the hide functionality is already implemented or not. The hide function is to be replaced